### ISSUE TITLE
keymapp error

### ISSUE THREAD

**Issue by llcc** [jkitchin/org-ref#5]
Dear John,

When I updated org-ref, emacs can not launch properly with error message "keymapp error".
Hope you can fix it. Thanks.

**Comment by jkitchin** [comment]
can you post the whole error? 

It works fine for me, and several others here. 

do you know if you have (require 'org) before you load org-ref?
recently l put this in which might cause that.
;; define key for inserting citations
(define-key org-mode-map
  (kbd org-ref-insert-cite-key)
  org-ref-insert-cite-function)

**Comment by llcc** [comment]
Thats very strange, i used -debug--init to launch emacs, but emacs only gave me 
"Wrong type argument: keymapp, nil". 

And it turns out i can org-babel-load-file org-ref without any error. I tried to add (require 'org) before org-ref, but no luck.

**Comment by jkitchin** [comment]
Maybe you need to install helm-bibtex? and maybe helm?

John

-----------------------------------
Professor John Kitchin
Doherty Hall A207F
Department of Chemical Engineering
Carnegie Mellon University
Pittsburgh, PA 15213
412-268-7803
@johnkitchin
http://kitchingroup.cheme.cmu.edu

**Comment by llcc** [comment]
I have helm-bibtex and helm installed. So very strange problem.

**Comment by jkitchin** [comment]
Are you still seeing this issue?

**Comment by llcc** [comment]
@jkitchin I changed the org-ref load-file position many times, and finally passed the launch inspection. However, because of plenties of codes in my config, I am sorry that I can not figure out what the problem is indeed. 

I have tried the new org-ref. It is gorgeous, thanks very much for giving us this package.

**Comment by jkitchin** [comment]
Great! Thanks for the update, and I am glad you like it!

**Comment by divinenephron** [comment]
I also produced this error.

**Steps to reproduce**

Check out a85b032e9f5083ec4d030e56486a7f70c72e0eb1.
Install the dependencies of `org-ref`.
Run `org-babel-tangle` on `org-ref.org` to produce `org-ref.el`.
Add the config from this [gist][1] to `init.el`.
Run `emacs --debug-init`.

**Result**

The backtrace produced by `emacs --debug-init` is included in the previous [gist][1]. My impression is `helm-bibtex` is trying to create menu items in the Helm menu, which does not exists because we have not `require`d `helm-config`.

[1]: https://gist.github.com/divinenephron/71fbf27d32afa848a405

**Workaround**

I added the following to `init.el`, which made the error disappear.

    (require 'helm-config)

**Fix**

This issue could be avoided in future if @jkitchin added a note to the configuration instructions making clear the workaround above is needed to run `org-ref`. Another solution that's more labour intensive would be for `org-ref` to use `helm-bibtex` only if it is already `require`d.
